website/content/en/blog/news/first-post/index.md

```diff
@@ -0,0 +1,46 @@
+---
+date: 2022-10-10
```

Comment:
We can keep only one page like an index of all the blog posts and workshops. since blogs will have their own intro and content, we will try to give 2-3 lines of description in general. What do you think?
(need - http://localhost:1313/kubeflow-manifests/main/blog/, dont need http://localhost:1313/kubeflow-manifests/main/blog/news/first-post/ for now)

Also, avoid introducing images since it increases the size of repository

Otherwise, placement etc look good